src/windows/osd.c

```diff
@@ -505,3 +505,43 @@ void freeifaddrs(struct ifaddrs *ifa)
 		ifa = next;
 	}
 }
+
+ssize_t ofi_sendmsg_socket(SOCKET fd, const struct msghdr *msg, int flags)
+{
+	ssize_t size;
+	int ret, i;
+	WSABUF *wsa_buf;
+
+	wsa_buf = (WSABUF *)calloc(msg->msg_iovlen, sizeof(WSABUF));
```

Comment:
Use alloca() instead, or see if it's possible to just cast from struct iovec to WSABUF (preferred)

Reply:
unfortunately the WSABUF has length first and address as second member while iovec has opposite.  I will change the stack allocation (almost created a mem leak here). Thanks for pointing it.